store/auth.go

```diff
@@ -200,9 +200,9 @@ func RequestPackageAccessMacaroon() (string, error) {
 	return responseData.Macaroon, nil
 }
 
-// RequestDischargeMacaroon requests a discharge macaroon for an existing macaroon
-func RequestDischargeMacaroon(username, password, macaroon, otp string) (string, error) {
-	const errorPrefix = "cannot get discharge macaroon: "
+// DischargeAuthCaveat returns a macaroon with the store auth caveat discharged
```

Comment:
s/$/./

Style hint: the comments should always look like proper English text, with punctuation paragraphs and all. We have several cases where that isn't true, but that's legacy which needs fixing.